### ISSUE TITLE
implementation details

### ISSUE THREAD

**Issue by argman** [Ziyan-Huang/FLARE24#1]
so its just a basic nnunet with different spacing and patch size ?

**Comment by Ziyan-Huang** [comment]
Fewer stages (4 vs. 6), more training epochs (2000 vs. 1000), and a lower initial learning rate (0.001 vs. 0.01) are also modified.

**Comment by argman** [comment]
OK, thanks!
